app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenPlayEditorFragment.java

```diff
@@ -20,7 +45,78 @@ public static ScreenPlayEditorFragment newInstance(Bundle args){
     }
 
     @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        screenPlayEditorManager = new ScreenPlayEditorManager();
+    }
+
+    @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         return inflater.inflate(R.layout.fragment_screenplay_editor,container,false);
     }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        slidePicture = (ImageView)view.findViewById(R.id.screenplay_slide_image);
+        final EditText slideDesc = (EditText)view.findViewById(R.id.editor_slide_text);
+        RecyclerView recyclerView = (RecyclerView)view.findViewById(R.id.screenplay_slide_list);
+        recyclerView.setHasFixedSize(true);
+        recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
+        SlideSelectorRecyclerAdapter slideSelectorRecyclerAdapter = new SlideSelectorRecyclerAdapter(screenPlayEditorManager);
+        recyclerView.setAdapter(slideSelectorRecyclerAdapter);
+
+        view.findViewById(R.id.editor_gallery).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                PictureUtils.pickPhotoFromGallery(ScreenPlayEditorFragment.this, REQUEST_CODE_GALLERY);
+            }
+        });
+
+        view.findViewById(R.id.editor_camera).setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                PictureUtils.takePhotoFromCamera(ScreenPlayEditorFragment.this, REQUEST_CODE_CAMERA);
+            }
+        });
+        screenPlayEditorManager.addSlide(new Slide("add_button", "add_button", 0));
```

Comment:
I need to put into adapter the add button, so I added the add button when I create adapter. Is it correct? or How do I put the element into adapter?